vm/src/builtins/iter.rs

```diff
@@ -13,6 +14,91 @@ use crate::{
     TypeProtocol,
 };
 
+#[derive(Debug)]
+pub struct PositionIterInternal {
+    pub position: AtomicCell<usize>,
```

Comment:
This should be `PyAtomic<usize>`, I think there's some examples of its use if you grep for it.